### ISSUE TITLE
dyld4::APIs::_dyld_missing_symbol_abort() when running in iOS Simulator

### ISSUE THREAD

**Issue by InvivoLuke** [RenderHeads/UnityPlugin-AVProVideo#1310]
**Describe the issue**

output shows
dyld[16866]: missing symbol called

and the stack track indicates 
dyld4::APIs::_dyld_missing_symbol_abort() 
AVPPluginBootstrap

on line 13 of AVProVideoBootstrap.m

**Your Setup (please complete the following information):**
 - Unity version: 2020.3.38
 - AVPro Video version (2.5.7 (ultra))
 - Operating system version: iOS 15.5 (Simulator)
 - Device model: iPad Pro
 - Video specs (resolution, frame-rate, codec, file size): 

**To Reproduce**
1. Build to iOS (using Simulator SDK)
2. Run

<img width="1141" alt="b7ba4e71-b49e-48ec-babd-95569afa4e92" src="https://user-images.githubusercontent.com/84787403/192938155-a7e4a927-198a-4fe9-a410-663444a9852b.png">

**Comment by MorrisRH** [comment]
Hey @InvivoLuke
I'm afraid we don't provide support for the iOS simulator with AVPro Video.

**Comment by InvivoLuke** [comment]
Thanks @MorrisRH ,

When was Simulator support dropped? It looks like it was added in 1.6.0..
https://forum.unity.com/threads/released-avpro-video-complete-video-playback-solution.385611/page-26#post-3011815

Would I be able to revert to an earlier version when it was supported? 

We're trying to automate our CI/CD pipeline and be able to generate screenshots for a number of iOS devices, would be ideal to use the Simulator instead of having to run tests on physical devices.

**Comment by MorrisRH** [comment]
It was dropped when we released version 2.0 back in January 2021. At that time the simulator didn't support Metal so we were unable to add it as a target (as with v2 we dropped support for OpenGL). A more recent complication came with the arrival of Apple Silicon which meant we were no longer able to embed the simulator code in the same plugin as the native device code.

There were some interface changes between v1 and v2 so you wouldn't be able to drop it in as is, we also no longer support it so there is a strong possibility it won't work as expected (or at all) anymore.

Issue #972 is open for adding the iOS simulator as an enhancement so if you'd like to register your interest there.
